.github/workflows/e2e-k8s.yaml

```diff
@@ -30,3 +30,7 @@ jobs:
     with:
       thin_csi_sanity: "true"
 
+  use-legacy:
+    uses: ./.github/workflows/e2e-k8s-latest-only-workflow.yaml
```

Comment:
https://github.com/topolvm/topolvm/blob/0239c1fb6a7631a0673f35f422ff542cc395a298/constants.go#L15-L17
OMG. This means that TopoLVM works in the legacy mode by passing environment variable `USE_LEGACY=false`. I've expected that TopoLVM works in the legacy mode _if and only if_ `USE_LEGACY=true`.

Reply:
Just beause it is time consuming.

Reply:
I'm considering moving the test for node finalizer from e2e to the middle test. This will reduce the number of jobs. After that, we can test the feature for all k8s versions.

Reply:
Then, I am not very happy about the large portion of `.github/workflows/e2e-k8s-latest-only-workflow.yaml` and `.github/workflows/e2e-k8s.yam`l that is duplicated.

Would it be possible to eliminate the duplication by, for example
1. change `e2e-k8s-latest-only-workflow.yaml` to a workflow that runs the e2e test for one k8s version received in input, instead of the latest k8s version,
2. in the e2e test for `USE_LEGACY=true`, call the workflow of 1. directly,
3. in `.github/workflows/e2e-k8s.yaml`, call the workflow of 1. with matrix strategy.

Reply:
Because the test for node finalizer mentioned in https://github.com/topolvm/topolvm/pull/665#discussion_r1124031783 is now going to be removed from e2e test by https://github.com/topolvm/topolvm/pull/670, I changed my mind and let legacy mode be tested with three Kubernetes versions.
Sorry for bothering you.